src/App.tsx

```diff
@@ -26,70 +27,106 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case 1:
```

Comment:
It's not clear what is `1` here - don't use an enum value in this way. 
```suggestion
    case SortType.ALPHABET:
```
fix it everywhere